docs/Release_Guide/metviewer_development.rst

```diff
@@ -7,5 +7,13 @@ Create a new vX.Y.Z-betaN or vX.Y.Z-rcN development release from the develop bra
 
 .. include:: release_steps/open_release_issue.rst
 .. include:: release_steps/clone_project_repository.rst
-.. include:: release_steps/checkout_main_branch.rst
-.. include:: release_steps/create_release_feature_branch.rst
+.. include:: release_steps/checkout_develop_branch.rst
+.. include:: release_steps/metviewer/update_version.rst
+.. include:: release_steps/update_release_notes_development.rst
+.. include:: release_steps/metviewer/update_docs_develop.rst
+.. include:: release_steps/merge_release_issue.rst
+.. include:: release_steps/create_release_on_github.rst
+.. include:: release_steps/metviewer/update_dtc_website.rst
+.. include:: release_steps/metviewer/create_release_extra.rst
```

Comment:
I think you want to move:
.. include:: release_steps/metviewer/create_release_extra.rst
to be right underneath:
.. include:: release_steps/create_release_on_github.rst
instead of having:
.. include:: release_steps/metviewer/update_dtc_website.rst
in between them.

Reply:
I moved suggested file and deleted update_dtc_website.rst since I don't perform this step